### DISCUSSION TITLE
Is there a way to auto optimize uploaded images

### DISCUSSION THREAD

**Discussion by alpap** [pocketbase/pocketbase#2485] [Q&A] [answered]
Is there a way to auto optimize uploaded images size wise ex 
* convert jpeg to png
* set quality

**Comment by ganigeorgiev** [comment, marked as answer]
No, there is no support for this out of the box.

Image files have only the following thumb size options available at the moment - https://pocketbase.io/docs/files-handling/#file-url.

**Reply by alpap** [reply]
thanks
